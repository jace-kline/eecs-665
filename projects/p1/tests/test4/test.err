FATAL [1,48]: Illegal character ?
