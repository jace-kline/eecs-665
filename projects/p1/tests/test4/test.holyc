"Can I put a single quote ' in a string literal?"