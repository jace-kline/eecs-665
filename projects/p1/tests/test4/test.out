ID:an [1,3]
ID:I [1,6]
ID:put [1,8]
ID:a [1,12]
ID:single [1,14]
ID:quote [1,21]
ID:in [1,29]
ID:a [1,32]
ID:string [1,34]
ID:literal [1,41]
EOF [1,49]
